src/main/java/factionsystem/Subsystems/StorageSubsystem.java

```diff
@@ -1,184 +1,197 @@
 package factionsystem.Subsystems;
 
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
 import factionsystem.Main;
 import factionsystem.Objects.ClaimedChunk;
 import factionsystem.Objects.Faction;
 import factionsystem.Objects.LockedBlock;
 import factionsystem.Objects.PlayerPowerRecord;
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.util.Scanner;
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.*;
 
 public class StorageSubsystem {
 
     Main main = null;
 
+    private final static String FILE_PATH = "./plugins/MedievalFactions/";
```

Comment:
You should add final!
```suggestion
    private final final static String FILE_PATH = "./plugins/MedievalFactions/";
```